docs/synapse/userguides/ug018_storm_ref_misc.rst

```diff
@@ -13,7 +13,54 @@ Todo
 
 limit()
 -------
-Todo
+Provides a hard limit on the number of nodes which are emitted by the ``limit()`` operator.
+This also back-propagates a ``limit=`` argument to the previous query operator.
+
+
+Optional parameters:
+
+* **Return limit:** specify the maximum number of nodes returned by the ``pivot()`` query.
+
+  * ``limit=`` (operator syntax)
+
+**Operator syntax:**
+
+.. parsed-literal::
+
+  **limit( *<limit>* )**
+
+**Macro syntax:**
+
+N/A
+
+**Examples:**
+
+* Limit the total number of nodes of a lift to 10 nodes:
+  ::
+    inet:ipv4*tag=hehe.haha limit(10)
+
+* Limit the number of nodes which were emitted by a ``refs()`` command to 10 nodes:
+  ::
+    inet:ipv4=8.8.8.8 refs() limit(10)
+
+* Limit the numbers of nodes which were emitted by a pivot macro operator to 10 nodes:
+  ::
+     inet:ipv4=8.8.8.8 inet:ipv4->inet:dns:a:ipv4 limit(10)
+
+* Perform a pivot, limiting the output with ``limit()``, then find all the tags which are on the output nodes:
+  ::
+     inet:ipv4=8.8.8.8 inet:ipv4->inet:dns:a:ipv4 limit(10) totags()
+
+* ``limit()`` will push its own limit into the preceeding query operator, so the following example would behave as if the ``pivot()`` had ``limit=10``:
```

Comment:
Would move this as discussion under "Usage notes" (see below)

Reply:
moved